engine/graphics/src/graphics.h

```diff
@@ -414,6 +414,17 @@ namespace dmGraphics
         uint32_t           m_ModificationVersion; // OpenGL
     };
 
+    struct Uniform
+    {
+        char*            m_Name; // Global/short name, e.g "my_member"
+        dmhash_t         m_NameHash;
+        char*            m_CanonicalName; // Full name, e.g "vsUniforms.some_struct.my_member"
```

Comment:
Do we really need both?
If I have a global "tint", what will the canonical name be? 🤔